torch/_inductor/kernel/mm.py

```diff
@@ -76,6 +76,8 @@
             b = tl.load(B, mask=rk[:, None] < k, other=0.)
         if B_PROLOGUE_CAST_TYPE is not None:
             b = b.to(B_PROLOGUE_CAST_TYPE)
+        {{prologue_A("a")}}
+        {{prologue_B("b")}}
```

Comment:
Should there be a flag on the template to say this is supported?  Other templates (like conv) won't support this.

Reply:
Yes, it is better to have a flag. Fixed by inserting a guard.